sdks/python/apache_beam/ml/inference/base.py

```diff
@@ -1477,26 +1638,31 @@ def __init__(
       model_handler: ModelHandler[ExampleT, PredictionT, Any],
       clock,
       metrics_namespace,
-      enable_side_input_loading: bool = False,
+      load_model_at_runtime: bool = False,
```

Comment:
This is pretty convenient functionality for large models too, good extra get here